t/subscription.t

```diff
@@ -8,12 +8,12 @@ my @received;
 
 package Example::Service {
     use Myriad::Service;
-    method simple_emitter : Emitter(
+    async method simple_emitter : Emitter(
```

Comment:
This doesn't return a Future, so it's not really async... we'd need to change the API if we wanted that.

Reply:
I'm getting this: `Too few arguments for subroutine 'Example::Service::simple_emitter' blib/lib/Myriad/Service/Implementation.pm line 159.` when I remove the async call.